setup.py

```diff
@@ -37,6 +38,7 @@
     "pyshacl==0.18.1",
     "gql==3.0.0",
     "aiohttp==3.8.1",
+    "redis==4.0.2",
```

Comment:
no need for redis and caching anymore, if we are not caching anything

Reply:
Forgot to delete it